targets/TARGET_ARM_SSG/TARGET_MUSCA_A1/device/COMPONENT_SPE/TOOLCHAIN_ARMC6/musca_s.sct

```diff
@@ -0,0 +1,211 @@
+#! armclang --target=arm-arm-none-eabi -march=armv8-m.main -E -xc
+
+/*
+ * Copyright (c) 2018 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "../../../partition/region_defs.h"
+
+#if !defined(TFM_LVL)
+  #define TFM_LVL   1
+#endif
+
+LR_CODE S_CODE_START {
+
+    /****  This initial section contains common code for TEE */
+    ER_TFM_CODE S_CODE_START S_CODE_SIZE {
```

Comment:
`S_CODE_START` and `S_CODE_SIZE` - From where are these values parsed?

Reply:
See [this answer](https://github.com/ARMmbed/mbed-os/pull/9221#discussion_r244934535)

Reply:
Shouldn't we use value from targets.json inherited from PSA_Target:
         "overrides": {
            "secure-rom-start": "xx",
            "secure-rom-size": "xx",
            "non-secure-rom-start": "xx",
            "non-secure-rom-size": "xx",
            "secure-ram-start": "xx",
            "secure-ram-size": "xx",
            "non-secure-ram-start": "xx",
            "non-secure-ram-size": "xx",
            "shared-ram-start": "xx",
            "shared-ram-size": "xx"
        }

Reply:
@jeromecoutant This comment is outdated and not relevant anymore.